.travis.yml

```diff
@@ -25,6 +25,13 @@ jobs:
         - BOTO_CONFIG: "/dev/null"
         - SO_ENABLE_MOTO_SERVER: "1"
         - TOXENV: "check_keys,py37-doctest,enable_moto_server,py37-test,py37-benchmark,py37-integration,disable_moto_server"
+    - python: '3.8'
```

Comment:
Please disable Py3.8 benchmarking for now.

I'm in the process of rewriting the benchmarks so we won't need a separate bucket for each Python version.

We also probably don't want to bother with the Moto server for all builds.

Reply:
So drop `py38-benchmark` and both the moto builds as well as the SO_ENABLE_MOTO_SERVER?

Reply:
Yes

Reply:
Got it. Thank you for the clarification.